airbyte-integrations/connector-templates/generator/build.gradle

```diff
@@ -14,7 +14,7 @@ task testScaffoldTemplates
 
 def addScaffoldTemplateTask(name, packageName,scaffoldParams=[]) {
     def taskName = "testScaffoldTemplate_${name}"
-    def outputDir = "airbyte-integrations/connectors/source-scaffold-${name}"
+    def outputDir = "airbyte-integrations/connectors/source-${packageName}"
```

Comment:
having `scaffold` in the name is helpful. Plus i don't think this was actually broken - see other comment